config/config.go

```diff
@@ -112,7 +112,7 @@ var (
 			MaxNumActsPerPool:  32000,
 			MaxGasLimitPerPool: 320000000,
 			MaxNumActsPerAcct:  2000,
-			ActionExpiry:       10 * time.Minute,
+			ActionExpiry:       time.Minute,
```

Comment:
with this PR, a transfer/exec with bad address won't pass validation, but will remain in actpool

thinking to reduce the action expiry time, to speed up removing the pending bad tx out of actpool

Reply:
1 min already translates to 12 chances of attempt (at 5 second a block)

if a tx failed for 1 min, it's meaningless to leave it 10 mins